lib/Github/Api/Repo.php

```diff
@@ -659,4 +659,24 @@ public function replaceTopics($username, $repository, array $topics)
 
         return $this->put('/repos/'.rawurldecode($username).'/'.rawurldecode($repository).'/topics', ['names' => $topics]);
     }
+
+    /**
+     * Transfer a repository.
+     *
+     * @link https://developer.github.com/v3/repos/#transfer-a-repository
+     *
+     * @param string $username
+     * @param string $repository
+     * @param string $new_owner
+     * @param array  $team_id
+     *
+     * @return array
+     */
+    public function transfer($username, $repository, $new_owner, $team_id = [])
```

Comment:
We should use camelCase and not snake_case

Reply:
@GrahamCampbell Is there any configuration in StyleCI that would allow us to force camel case?